src/services/pcn-router/src/Ports.h

```diff
@@ -16,88 +16,67 @@
 
 #pragma once
 
-#include "../interface/PortsInterface.h"
+#include "../base/PortsBase.h"
 
-#include "polycube/services/cube.h"
-#include "polycube/services/port.h"
-#include "polycube/services/utils.h"
-
-#include <spdlog/spdlog.h>
 #include <set>
 
 #include "PortsSecondaryip.h"
 
 class Router;
 
-#define SECONDARY_ADDRESS 5
+#define MAX_SECONDARY_ADDRESSES 5
```

Comment:
This constant is defined twice.
Either we defined only in one place, or we add a comment (in both places) telling the programmer that this constant has to be updated in both places.